doc/api/vm.markdown

```diff
@@ -41,6 +41,10 @@ e.g. `(0,eval)('code')`. However, it also has the following additional options:
 
 - `filename`: allows you to control the filename that shows up in any stack
   traces produced.
+- `lineOffset`: allows you to add an offset to the line number that is
```

Comment:
I've updated the docs in two places with the lineOffset and columnOffset parameters, and also added a missing filename and timeout parameter listings.